prog/vm/nexus.rb

```diff
@@ -239,7 +237,7 @@ def before_run
     case host.sshable.cmd("common/bin/daemonizer --check prep_#{q_vm}")
     when "Succeeded"
       host.sshable.cmd("common/bin/daemonizer --clean prep_#{q_vm}")
-      vm.nics.each { _1.incr_setup_nic }
+      vm.private_subnets.map(&:incr_add_new_nic)
```

Comment:
nit: we don't use output
```suggestion
      vm.private_subnets.each(&:incr_add_new_nic)
```